prow/plugins/trigger/pull-request.go

```diff
@@ -286,8 +286,18 @@ I understand the commands that are listed [here](https://go.k8s.io/bot-commands?
 %s
 </details>
 `, author, org, org, more, joinOrgURL, labels.OkToTest, encodedRepoFullName, plugins.AboutThisBotWithoutCommands)
-		if err := ghc.AddLabel(org, repo, pr.Number, labels.NeedsOkToTest); err != nil {
+
+		l, err := ghc.GetIssueLabels(org, repo, pr.Number)
```

Comment:
Nit: Could we remove the empty line between this and the err checl?

Reply:
Fixed as well :+1: